### PR TITLE
DF-160: Add s3 image in notification table

### PR DESCRIPTION
**Pull request by AyushGiri190**
image
image

### CODE REVIEW

File: src/components/Notification/addNotify.tsx [hunk 1]
```diff
@@ -51,86 +52,133 @@ const faqSchema = z.object({
...
-    console.log("Form data:", data); // Log data to verify it's being captured
+  const handleDrop = (event: React.DragEvent<HTMLDivElement>) => {
+    event.preventDefault();
+    const file = event.dataTransfer.files?.[0];
+    if (file && allowedImageFormats.includes(file.type)) {
+      if (file.size <= maxImageSize) {
+        setSelectedPicture(file);
+        setPreviewUrl(URL.createObjectURL(file));
+      } else {
+        alert('File size exceeds the 2MB limit.');
```
Comment (Divesh-singh) [inline src/components/Notification/addNotify.tsx:124]: here too

File: src/components/Notification/addNotify.tsx [hunk 2]
```diff
@@ -51,86 +52,133 @@ const faqSchema = z.object({
...
+       formData,
         {
           headers: {
-            "Content-Type": "multipart/form-data",
+            'Content-Type': 'multipart/form-data',
           },
         },
       );
-      reset();
+      const { Location } = postResponse.data.data;
```
Comment (Divesh-singh) [inline src/components/Notification/addNotify.tsx:155]: add null checks

File: src/components/Notification/notifyTable.tsx [hunk 3]
```diff
@@ -55,22 +55,22 @@ const NotifyTable: React.FC = () => {
   const [userData, setUserData] = useState<UserData[]>([]);
   const [loading, setLoading] = useState(true);
   const { toast } = useToast();
-  useEffect(() => {
+  
     const fetchUserData = async () => {
       try {
         const response = await apiHelperService.getAllNotification();
-        setUserData(response.data.data);
+        setUserData(response.data.data||[]);
```
Comment (Divesh-singh) [inline src/components/Notification/notifyTable.tsx:62]: add null checks here

**Review by Divesh-singh — CHANGES_REQUESTED** [review]
address comments

File: src/components/Notification/addNotify.tsx [hunk 4]
```diff
@@ -121,7 +123,7 @@ const fileInputRef = useRef<HTMLInputElement | null>(null);
         setSelectedPicture(file);
         setPreviewUrl(URL.createObjectURL(file));
       } else {
-        alert('File size exceeds the 2MB limit.');
+        alert(`File size exceeds the ${sizeInMb} limit.`);
```
Comment (Divesh-singh) [inline src/components/Notification/addNotify.tsx:126]: Add MB after placeholder in alert

**Comment by Divesh-singh** [comment]
Resolve conflict

**Comment by AyushGiri190** [comment]
> Resolve conflict
resolved

**Pull request merged**